dev-python/QtPy/QtPy-1.9.0-r2.ebuild

```diff
@@ -30,7 +30,7 @@ RDEPEND="
 BDEPEND="
 	test? (
 		dev-python/mock[${PYTHON_USEDEP}]
-		dev-python/PyQt5[${PYTHON_USEDEP},bluetooth,dbus,declarative,designer,gui,help,location,multimedia,network,networkauth,opengl,positioning,printsupport,sensors,serialport,sql,ssl,svg,testlib,webchannel,webkit,websockets,widgets,x11extras,xml(+),xmlpatterns]
+		dev-python/PyQt5[${PYTHON_USEDEP},bluetooth,dbus,declarative,designer,gui,help,location,multimedia,network,opengl,positioning,printsupport,sensors,serialport,sql,ssl,svg,testlib,webchannel,websockets,widgets,x11extras,xml(+),xmlpatterns]
```

Comment:
I dare say you need to revbump. Otherwise, people who have installed the old version already won't be able to upgrade PyQt5.